tests/Equinox.Templates.Tests/DotnetBuild.fs

```diff
@@ -75,6 +75,8 @@ type DotnetBuild(output : ITestOutputHelper, folder : EquinoxTemplatesFixture) =
     let [<Fact>] proReactorDefault ()           = run "proReactor" []
     let [<Fact>] proReactorFilter ()            = run "proReactor" ["--filter"]
 
+    let [<Fact>] proCosmosReactor args          = run "proCosmosReactor" args
```

Comment:
```suggestion
    let [<Fact>] proCosmosReactor ()          = run "proCosmosReactor" []
```

Reply:
Thanks! It works! Pushing new version up.